wiki/Twitter/es.md

```diff
@@ -15,11 +15,11 @@ osu! tiene bastantes cuentas de Twitter, cada una para diferentes propósitos. L
 | :-: | :-: | :-- |
 | ![osu! avatar](img/osugame.jpg) | [@osugame](https://twitter.com/osugame "Twitter") | La fuente oficial de noticias y anuncios. |
 | ![BanchoBoat avatar](img/banchoboat.jpg) | [@banchoboat](https://twitter.com/banchoboat "Twitter") | Chistes cuando las cosas salen mal. |
-| ![NAT avatar](img/osu_nat.png) | [@osu_nat](https://twitter.com/osu_nat "Twitter") | Noticias, anuncios, y breves encuestas comunitarias realizadas por el [NAT](/wiki/People/Nomination_Assessment_Team) (no manejado por [el osu!team](/wiki/People/osu!_team)). |
-| ![pp committee avatar](img/ppcommittee.png) | [@pp_committee](https://twitter.com/pp_committee) | Anuncios de cálculo de dificultad para todos los modos de juego, manejado por el  [Comité de Puntos de Rendimiento](/wiki/People/Performance_Points_Committee). |
+| ![NAT avatar](img/osu_nat.png) | [@osu_nat](https://twitter.com/osu_nat "Twitter") | Noticias, anuncios, y breves encuestas comunitarias realizadas por el [NAT](/wiki/People/Nomination_Assessment_Team) (no manejado por [el osu! team](/wiki/People/osu!_team)). |
+| ![pp committee avatar](img/ppcommittee.png) | [@pp_committee](https://twitter.com/pp_committee) | Anuncios de cálculo de dificultad para todos los modos de juego, manejado por el [comité de puntos de rendimiento](/wiki/People/Performance_Points_Committee). |
 
 ## Personal
 
 | Avatar | Usuario | Descripción |
 | :-: | :-: | :-- |
-| ![Dean Herbert avatar](img/ppy.jpg?2) | [@ppy](https://twitter.com/ppy "Twitter") | La cuenta personal del [creador de osu!](/wiki/People/peppy), que no es estrictamente sobre osu!, pero de todas maneras cubre la mayoría de ella. |
+| ![Dean Herbert avatar](img/ppy.jpg?2) | [@ppy](https://twitter.com/ppy "Twitter") | La cuenta personal de Twitter del [creador de osu!](/wiki/People/peppy), que no es estrictamente sobre osu!, pero de todas maneras cubre la mayoría de ella. |
```

Comment:
In the original article, the last `it` is making reference to the game, not the account. If we're implicitly going to refer to it (the game), it'd be more correct if the correct gendered pronoun was used (El Juego, not La Juego).
```suggestion
| ![Dean Herbert avatar](img/ppy.jpg?2) | [@ppy](https://twitter.com/ppy "Twitter") | La cuenta personal de Twitter del [creador de osu!](/wiki/People/peppy), que no es estrictamente sobre osu!, pero de todas maneras cubre la mayoría de él. |
```